config/config.go

```diff
@@ -431,6 +431,38 @@ type HealthConfig struct {
 	Rate []Rate `yaml:"rate,omitempty" json:"rate,omitempty"`
 }
 
+// Versions version ranges
+type Versions struct {
+	MeshName     string `yaml:"meshName"`
+	VersionRange []struct {
+		MeshVersion string `yaml:"meshVersion"`
+		KialiLow    string `yaml:"kialiLow"`
+		KialiHigh   string `yaml:"kialiHigh"`
+	} `yaml:"versionRange"`
+}
+
+// NewVersions return compatible kiali versions for mesh
+func NewVersions() (*Versions, error) {
```

Comment:
Failing to check for version compatibility doesn't mean Kiali won't work - it just means it doesn't know (just like today :-).

We should not return an error (or at least we should not abort on any error) here. Just log a Warning message like you do and keep going. Kiali should still run as before.

Reply:
Yes, checking version should not impact Kiali running status. I catch the error for logging warning messages.